kernel/aster-nix/src/syscall/madvise.rs

```diff
@@ -33,8 +33,17 @@ pub fn sys_madvise(
 }
 
 fn madv_free(start: Vaddr, len: usize, ctx: &Context) -> Result<()> {
-    debug_assert!(start % PAGE_SIZE == 0);
-    debug_assert!(len % PAGE_SIZE == 0);
+    if start % PAGE_SIZE != 0 {
+        return_errno_with_message!(Errno::EINVAL, "the start address should be page aligned");
+    }
+
+    if len % PAGE_SIZE != 0 {
+        return_errno_with_message!(Errno::EINVAL, "the length should be page aligned");
+    }
+
```

Comment:
Could you remove the two blank lines between the three code blocks for validating arguments to make the code more compact? They should be combined in a single code paragraph without blank line because they serve a single purpose, which is to check if the arguments are valid or not.